.github/workflows/deploy-site.yaml

```diff
@@ -20,6 +20,10 @@ jobs:
     environment: website
     runs-on: ubuntu-latest
 
+    concurrency:
```

Comment:
Can we move this to the workflow leve rather than the job level?

Reply:
Yep, done